admin-dev/themes/new-theme/js/pages/order/view/order-product-edit.js

```diff
@@ -127,7 +134,6 @@ export default class OrderProductEdit {
 
     // Init editor data
     this.taxRate = product.tax_rate;
-    this.totalQuantity = product.availableQuantity + product.quantity;
```

Comment:
@zuk3975 Expected ? No rebase mistake ?

Reply:
It was intentional. Check lines (-47) and 49. As I remember it had something to do with dynamic stock value checking instead of product.availableQuantity (or something like that, i don't remember)